pages/docs/create.md

```diff
@@ -236,7 +236,7 @@ type User struct {
   FirstName string
   LastName  string
   Age       uint8
-  FullName  string `gorm:"->;type:GENERATED ALWAYS AS (concat(firstname,' ',lastname));default:(-);`
+  FullName  string `gorm:"->;type:GENERATED ALWAYS AS (concat(firstname,' ',lastname));default:(-);"`
```

Comment:
i good with running with cockroachdb:
by writting:
```suggestion
  FullName  string `gorm:"->;type:text GENERATED ALWAYS AS (concat(firstname,' ',lastname));default:(-);"`
```

@jinzhu  is it in every database so? - should i write another PR?

Reply:
the text is not necessary?